cli_tools/gce_vm_image_import/main.go

```diff
@@ -141,10 +148,32 @@ func validateAndParseFlags() error {
 			return err
 		}
 	}
+
 	return nil
 }
 
-//Returns main workflow and translate workflow paths (if any)
+// Validate source file is not a compression file by checking file header.
+func validateSourceFile(storageClient commondomain.StorageClientInterface) (error) {
+	if sourceFile == nil {
+		return nil
+	}
+
+	rc, err := storageClient.GetObjectReader(*sourceBucketName, *sourceObjectName)
+	if err != nil {
+		return fmt.Errorf("readFile: unable to open file from bucket %q, file %q: %v", *sourceBucketName, *sourceObjectName, err)
+	}
+	defer rc.Close()
+
+	// Try to extract compression file header of the source file.
+	_, err = gzip.NewReader(rc)
+	if err == nil {
+		return fmt.Errorf("source file is a compression file. You may try 'image create' instead of 'image import'")
```

Comment:
```suggestion
		return fmt.Errorf("source file is a compressed file. Please use 'images create' instead of 'images import'")
```

Reply:
Yes, any error thrown from gce_vm_image_import (or any other cli tools) will be natually adopted by gcloud.